source/mainnet/net/guides/recover-wallet.rst

```diff
@@ -13,6 +13,21 @@ In case you get a new device or need to restore your existing device, you can re
 
 .. tabs::
```

Comment:
![Screenshot from 2022-09-29 16-08-57](https://user-images.githubusercontent.com/87635671/193054628-6fa571c3-39ea-44e4-83ce-3e63ff3b1684.png)

Is this the intention, that both menus are open/selected here? :thinking: